### DISCUSSION TITLE
Docker - Image or docker-compose

### DISCUSSION THREAD

**Discussion by pablorq** [wintercms/winter#92] [Ideas]
It could be nice to have a docker image or at least a docker-compose file to easy install.

Keep going!

**Comment by bennothommo** [comment]
@pablorq - @mik-p has you covered for the moment: https://github.com/mik-p/docker-wintercms :)

**Reply by mik-p** [reply]
There are some docker compose examples in the readme on git (or there was until the automation blew them away, will fix)

There are some images available at -> https://hub.docker.com/r/hiltonbanes/wintercms

The update stuff is totally broken. I am working on it this week

**Reply by pablorq** [reply]
Nice to hear that!

Maybe this url (https://github.com/mik-p/docker-wintercms) can be added to the Readme on wintercms/winter, because right now there is a link on **Installing Winter** that points to https://wintercms.com/docs/setup/installation, but the web is redirecting to the homepage and the docs can't be viewed.

I'll try it later.

Regards!

**Reply by mik-p** [reply]
I changed the container update script to track wintercms' latest github releases. this appears to be working now. still waiting on the marketplace so 'winter plugin:install' command can be used in dockerfiles - hopefully this will be possible soon

**Reply by LeMaX10** [reply]
Support Laravel sail for October or Winter - https://packagist.org/packages/lemax10/asail
